tensorflow/c/kernels.h

```diff
@@ -65,6 +66,10 @@ typedef struct TF_KernelBuilder TF_KernelBuilder;
 typedef struct TF_OpKernelConstruction TF_OpKernelConstruction;
 typedef struct TF_OpKernelContext TF_OpKernelContext;
 
+// TF_InitKernel to do op/kernel registration.
+// Plugin needs to implement this function to register all kernels.
```

Comment:
Kernels can be registered using static registration as well (as it is done here: https://cs.opensource.google/tensorflow/tensorflow/+/master:tensorflow/c/kernels/ops/bitcast.cc;l=127?q=tensorflow%2Fc%2F%20bitcast). Therefore, "need" seems too strong of a word.

Change description to:
Kernel plugins should either implement `TF_InitKernel` to register kernels or use static registration. This function should register all kernels in a plugin.

Reply:
done